lib/commands/certificate.js

```diff
@@ -409,6 +410,26 @@ commands.mobileInstallCertificate = async function mobileInstallCertificate (opt
   }
 };
 
+/**
+ * Lists installed certificates for real devices only
+ * if [py-ios-device](https://github.com/YueChen-C/py-ios-device) tool
+ * is available on the server machine.
+ *
+ * @returns {Object} Returns map of certificates installed on the real device
+ * @throws {Error} If attempting to list certificates for simulated device or if py-ios-device
+ * is not installed
+ */
+commands.mobileListCertificates = async function mobileListCertificates () {
+  if (!this.isRealDevice()) {
+    throw new errors.NotImplementedError(`This extension is only supported on real devices`);
```

Comment:
Backticks are not needed here as the string does not contain any template variables. Vanilla single quotes would do

Reply:
Changed backticks to single quotes for both errors